storage/plugins/archives/ArchivesPlugin.php

```
@@ -3983,6 +3995,58 @@ public function plannedHooks(): array
+     * the first argument is the accumulated $results array; subsequent
+     * positional arguments come from the $args array passed to Hooks::apply().
+     *
+     * @param list<array<string, mixed>> $results
+     * @return list<array<string, mixed>>
+     */
+    public function addArchivalSources(array $results, string $q): array
+    {
+        if ($q === '') {
+            return $results;
+        }
+        $s    = '%' . $q . '%';
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $s. Configured minimum length is 3. (undefined)

(ShortVariable)